shells/firefox/build.js

```diff
@@ -15,11 +15,15 @@ const main = async () => {
     join(__dirname, 'build')
   );
 
-  console.log('\nThe Firefox extension has been built!');
+  console.log(chalk.green('\nThe Firefox extension has been built!'));
+  console.log(chalk.green('You can test it by following the steps below:'));
   console.log(chalk.gray('\n# Go to the unpacked directory:'));
   console.log(`cd ${unpackedPath}`);
   console.log(chalk.gray('\n# And launch Firefox with the extension enabled:'));
+  console.log('web-ext run');
+  console.log(chalk.gray('\n# You can also test against specific Firefox versions:'));
   console.log('web-ext run --firefox=nightly');
+  console.log('web-ext run --firefox=/Applications/Firefox54.app/Contents/MacOS/firefox-bin');
```

Comment:
I updated the README as well as the on-screen instructions.

![screen shot 2017-06-08 at 8 35 01 am](https://user-images.githubusercontent.com/29597/26937127-657eebbc-4c25-11e7-986b-c758c59be51e.png)